src/test/java/com/wiley/autotest/framework/tests/repeater/RepeaterPositiveTest.java

```diff
@@ -0,0 +1,18 @@
+package com.wiley.autotest.framework.tests.repeater;
+
+import com.wiley.autotest.framework.config.BaseTest;
+import com.wiley.autotest.framework.pages.RepeaterPage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.testng.annotations.Test;
+
+public class RepeaterPositiveTest extends BaseTest {
```

Comment:
please combine ALL tests for Repeater in 1 test class. (delete this one)

Reply:
Done